src/main/java/shizuka/Deadline.java

```diff
@@ -0,0 +1,15 @@
+package shizuka;
+
+public class Deadline extends Todo {
+    private String deadline;
+
+    public Deadline(String[] args) {
+        super(args[0]);
+        this.taskType = "D";
```

Comment:
Perhaps you can avoid the unnecessary use of `this`. Similar for all occurrences.

https://se-education.org/guides/conventions/java/index.html#variables